validation/valid/ttml2-valid-font-embedding-family-case-sensitive.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- valid - tt:font - @family - "near" use of generic family name, were it not for case sensitive comparison -->
+<tt xml:lang="" xmlns="http://www.w3.org/ns/ttml"
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter" ttp:profile="ttml2-transformation"
+    xmlns:ttva="http://skynav.com/ns/ttv/annotations" ttva:warnOn="all" ttva:expectedErrors="0" ttva:expectedWarnings="0" ttva:model="ttml2">
+  <head>
+    <resources>
+      <font src="../resources/test.otf" type="font/otf" family="Default"/>
+      <font src="../resources/test.otf" type="font/otf" family="Monospace"/>
+      <font src="../resources/test.otf" type="font/otf" family="SansSerif"/>
+      <font src="../resources/test.otf" type="font/otf" family="Serif"/>
+      <font src="../resources/test.otf" type="font/otf" family="MonospaceSansSerif"/>
+      <font src="../resources/test.otf" type="font/otf" family="MonospaceSerif"/>
+      <font src="../resources/test.otf" type="font/otf" family="ProportionalSansSerif"/>
+      <font src="../resources/test.otf" type="font/otf" family="ProportionalSerif"/>
```

Comment:
We may as well reduce the scope for over-eager validation errors here and reference font resources that do exist in the repo (noting that some of the referenced fonts don't match the family that they apparently are supposed to, because we don't have any monospaced fonts in the repo at the moment):

```suggestion
      <font src="../resources/fonts/NotoSans-Regular.ttf" type="font/otf" family="Default"/>
      <font src="../resources/fonts/NotoSans-Regular.ttf" type="font/otf" family="Monospace"/>
      <font src="../resources/fonts/NotoSans-Regular.ttf" type="font/otf" family="SansSerif"/>
      <font src="../resources/fonts/ScheherazadeRegOT.ttf" type="font/otf" family="Serif"/>
      <font src="../resources/fonts/NotoSans-Regular.ttf" type="font/otf" family="MonospaceSansSerif"/>
      <font src="../resources/fonts/ScheherazadeRegOT.ttf" type="font/otf" family="MonospaceSerif"/>
      <font src="../resources/fonts/NotoSans-Regular.ttf" type="font/otf" family="ProportionalSansSerif"/>
      <font src="../resources/fonts/ScheherazadeRegOT.ttf" type="font/otf" family="ProportionalSerif"/>
```

Reply:
"../resources/test.otf" does exist in the validation subdirectory, just not in the presentation subdirectory; also, the family name (in the `font` element) overrides any family name in the font so it doesn't matter; however, we do need to add `type="font/ttf"` to each of these `font` elements

Reply:
Ah, sorry, I was indeed looking in the wrong resources directory.